apps/frontend/nginx.conf

```diff
@@ -1,20 +0,0 @@
-server {
-  listen 80;
-
-  gzip on;
-  gzip_disable "msie6";
-
-  gzip_vary on;
-  gzip_proxied no-cache no-store private expired auth;
-  gzip_buffers 16 8k;
-  gzip_http_version 1.1;
-  gzip_types text/plain application/javascript text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/vnd.ms-fontobject application/x-font-ttf font/opentype;
```

Comment:
The new config file doesn't have the same gzip config. This was added just over a month ago to improve performance (#450). Does the unprivileged image enable this by default, or should this also be copied over too?

Reply:
Thank you for your suggestion.
I checked the nginx-unprivileged image and it does not have the "gzip" settings, so I added them back.